src/tests/about.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import About from '../pages/About/About';
+
+describe('testing about page', () => {
```

Comment:
The word `testing` is redundant here. Just `about page` is enough.

Reply:
Removed.